src/test/java/com/commercetools/project/sync/model/ResourceIdsGraphQlRequestTest.java

```diff
@@ -7,6 +7,7 @@
 import com.commercetools.sync.commons.helpers.ResourceKeyIdGraphQlRequest;
```

Comment:
Why we are testing utilities that are part of the sync-java project?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/268/commits/886c10342420e8c8c24720017869fda4cfc83d41